sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -58,34 +58,40 @@ az cognitiveservices account create \
     --location westus2 \
     --yes
 ```
-### Authenticate the client
-In order to interact with the Text Analytics service, you will need to create an instance of the `TextAnalyticsClient` 
-class. You will need an **endpoint** and either an **API key** or **AAD TokenCredential** to instantiate a client 
-object. And they can be found in the [Azure Portal][azure_portal] under the "Quickstart" in your created
-Text Analytics resource. See the full details regarding [authentication][authentication] of Cognitive Services.
+In order to interact with the Text Analytics service, you will need to create an instance of the Text Analytics client,
+both the asynchronous and synchronous clients can be created by using `TextAnalyticsClientBuilder` invoking `buildClient()`
+creates a synchronous client while `buildAsyncClient()` creates its asynchronous counterpart.
 
-#### Get credentials
-The authentication credential may be provided as the API key to your resource or as a token from Azure Active Directory.
+You will need an **endpoint** and either an **key** or **AAD TokenCredential** to instantiate a client object. 
 
+##### Looking up the endpoint
+You can find the **endpoint** for your Text Analytics resource in the [Azure Portal][azure_portal] under the "Keys and Endpoint",
+or [Azure CLI][azure_cli_endpoint].
+```bash
+# Get the endpoint for the text analytics resource
+az cognitiveservices account show --name "resource-name" --resource-group "resource-group-name" --query "endpoint"
+```
 
-##### **Option 1**: Create TextAnalyticsClient with AzureKeyCredential
-To use AzureKeyCredential authentication, provide the [key][key] as a string to the [AzureKeyCredential][azure_key_credential]. This can be found in the [Azure Portal][azure_portal] 
-   under the "Quickstart" section or by running the following Azure CLI command:
+##### Create a Text Analytics client with key credential
```

Comment:
with AzureKeyCredential?

Reply:
Since we mentioned key or AAD in the summary section:
```
You will need an **endpoint** and either an **key** or **AAD TokenCredential** to instantiate a client object. 
```

I think it is easier for user to refer to key credential other than mentioned a java object type directly here.